<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorCurrent</class>
 <widget class="QWidget" name="MotorCurrent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QDoubleSpinBox" name="spinDriver">
     <property name="toolTip">
      <string>Drive current</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string>A</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinIdle">
     <property name="toolTip">
      <string>Idle current</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string>A</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinSwitchTime">
     <property name="toolTip">
      <string>Driver and idle switch time</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
